go/prometheusexporter/collector/collector.go

```diff
@@ -473,33 +612,63 @@ func (c *Collector) deleteLabels(name string, labels prometheus.Labels) bool {
 	deleted := false
 	switch name {
 	case "logjam:action:http_response_time_summary_seconds":
-		deleted = c.httpRequestSummaryVec.Delete(labels)
+		deleted = c.actionMetrics.httpRequestSummaryVec.Delete(labels)
 	case "logjam:action:http_requests_total":
-		deleted = c.httpRequestsTotalVec.Delete(labels)
+		deleted = c.actionMetrics.httpRequestsTotalVec.Delete(labels)
 	case "logjam:action:job_execution_time_summary_seconds":
-		deleted = c.jobExecutionSummaryVec.Delete(labels)
+		deleted = c.actionMetrics.jobExecutionSummaryVec.Delete(labels)
 	case "logjam:action:job_executions_total":
-		deleted = c.jobExecutionsTotalVec.Delete(labels)
+		deleted = c.actionMetrics.jobExecutionsTotalVec.Delete(labels)
 	case "logjam:action:http_response_time_distribution_seconds":
-		deleted = c.httpRequestHistogramVec.Delete(labels)
+		deleted = c.actionMetrics.httpRequestHistogramVec.Delete(labels)
 	case "logjam:action:job_execution_time_distribution_seconds":
-		deleted = c.jobExecutionHistogramVec.Delete(labels)
+		deleted = c.actionMetrics.jobExecutionHistogramVec.Delete(labels)
 	case "logjam:action:page_time_distribution_seconds":
-		deleted = c.pageHistogramVec.Delete(labels)
+		deleted = c.actionMetrics.pageHistogramVec.Delete(labels)
 	case "logjam:action:ajax_time_distribution_seconds":
-		deleted = c.ajaxHistogramVec.Delete(labels)
-	case "logjam:action:action_calls_total":
-		deleted = c.actionCallsTotalVec.Delete(labels)
+		deleted = c.actionMetrics.ajaxHistogramVec.Delete(labels)
+	case "logjam:action:transactions_total":
+		deleted = c.actionMetrics.transactionsTotalVec.Delete(labels)
+	case "logjam:application:http_response_time_summary_seconds":
```

Comment:
I don't think adding the `logjam:application:` metrics here will have any effect.
`labels` always contains `"action"`, which means that `Delete(labels)` on an application metric will never match a time-series and effectively be a no-op.

Reply:
you're right, this code is superfluous. I got rid of it.